cmd/protoc-gen-cpp-tableau-loader/embed/hub.pc.cc

```diff
@@ -320,10 +324,12 @@ bool LoadMessageWithPatch(google::protobuf::Message& msg, const std::string& pat
       break;
     }
     case tableau::PATCH_MERGE: {
-      // load msg from the "main" file
-      if (!LoadMessageByPath(msg, path, fmt, options)) {
-        return false;
-      }
+      if (!options || options->load_mode != LoadMode::kModeOnlyPatch) {
```

Comment:
No need to check `options` nullity, as code **Line 289** in this func has already checked.